code/src/App.js

```diff
@@ -1,9 +1,52 @@
-import React from 'react'
+import React from 'react';
+import { Provider } from 'react-redux';
+import { combineReducers, createStore } from '@reduxjs/toolkit';
+// removed configureStore
```

Comment:
Having a sort of "history" in the comments in order to know what you removed is a good idea. Hadn't thought of it!